### ISSUE TITLE
.on('update') not always triggered

### ISSUE THREAD

**Issue by thmsbgg** [inwaar/gree-hvac-client#10]
Hi!

First, thank you a lot for your work! 

So about the issue, the 'update' event is only triggered when I connect but after when I use setProperty or setProperties the event is never triggered (the property is set correctly on my Gree hvac).

But if I use the official app, the event is triggered.

Am I missing something ?

**Comment by inwaar** [comment]
Hi thmsbgg, is the `update` event triggered if you change properties by the AC remote control?

You can listen for the `success` event to check if the properties you change were successfully changed.

**Comment by thmsbgg** [comment]
So yes the update is triggered if I change properties by the AC remote control or by the Android App.

Indeed if I listen to the `success` event I get the properties successfully changed, thanks!

But is it the normal behaviour to get the `update` event only when I change a property from an external remote device only ?

**Comment by inwaar** [comment]
Yes, if you want to get all events just subscribe for both `update` and `success`.
This needs to be added to the documentation and examples.

**Comment by thmsbgg** [comment]
Thanks a lot for your help : )
